pkg/converter/namespace_converter.go

```diff
@@ -15,12 +15,10 @@
 package converter
 
 import (
-	"fmt"
-	"reflect"
-
 	"github.com/projectcalico/libcalico-go/lib/api"
-	log "github.com/sirupsen/logrus"
-	k8sApiV1 "k8s.io/client-go/pkg/api/v1"
+	"github.com/projectcalico/libcalico-go/lib/backend/k8s"
+	backendConverter "github.com/projectcalico/libcalico-go/lib/converter"
+	"k8s.io/client-go/pkg/api/v1"
```

Comment:
ugh, this is going to migrate to k8s.io/app/core/v1 soon.

Reply:
Yep.. we're going to have a bunch of stuff to update here and libcalico and perhaps elsewhere... :/